components/authz-service/server/v2/rules_property_test.go

```diff
@@ -18,17 +21,511 @@ import (
 )
 
 const (
-	idRegex = "^[a-z0-9-]{1,64}$"
+	idRegex        = "^[a-z0-9-]{1,64}$"
+	conditionLimit = 10 // avoid the grpc limit of 4194304
```

Comment:
As mentioned in the PR preamble, ListRules could exceed GRPC capacity if the number of rule conditions was not constrained -- which could be a problem in the real world.